src/sidebar/helpers/version-data.ts

```diff
@@ -1,22 +1,30 @@
-/**
- * @typedef {import('../../types/annotator').SegmentInfo} SegmentInfo
- * @typedef {import('../store/modules/frames').Frame} Frame
- */
+import type { SegmentInfo } from '../../types/annotator';
+import type { Frame } from '../store/modules/frames';
 
-/**
- * @typedef UserDetails
- * @prop {string|null} [userid]
- * @prop {string} [displayName]
- */
+type UserDetails = {
+  userid?: string | null;
+  displayName?: string;
+};
 
 export class VersionData {
+  public version: string;
```

Comment:
The `public` modifier here is not strictly required since fields are public by default. We haven't settled on an official convention yet. With methods though the pattern we've been following is to add a `private` modifier before private methods and leave off the modifier before public methods. I'm inclined to do the same with fields for consistency. We can discuss in more depth tomorrow.

Reply:
True. Good catch.